service/history/workflowcache/cache.go

```diff
@@ -97,6 +124,11 @@ func (c *wfCache) AllowExternal(domainID string, workflowID string) bool {
 
 // AllowInternal returns true if the rate limiter for this domain/workflow allows an internal request
 func (c *wfCache) AllowInternal(domainID string, workflowID string) bool {
+	if !c.workflowIdCacheEnabledCheck(domainID) {
+		// If we can't get the cache item, we should allow the request through
```

Comment:
nit: match the comment when we allow requests for internal and external requests because cache is not enabled

Reply:
Thanks, good catch!